x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_lists.test.tsx

```diff
@@ -141,9 +201,11 @@ describe('AutocompleteFieldListsComponent', () => {
       </ThemeProvider>
     );
 
-    ((wrapper.find(EuiComboBox).props() as unknown) as {
-      onChange: (a: EuiComboBoxOptionOption[]) => void;
-    }).onChange([{ label: 'some name' }]);
+    await act(async () => {
+      ((wrapper.find(EuiComboBox).props() as unknown) as {
+        onChange: (a: EuiComboBoxOptionOption[]) => void;
+      }).onChange([{ label: 'some name' }]);
+    });
 
     expect(mockOnChange).toHaveBeenCalledWith({
       created_at: '2020-04-20T15:25:31.830Z',
```

Comment:
nit: some of these are constants such as this one that is `DATE_NOW` located here:

```ts
x-pack/plugins/lists/common/constants.mock.ts
```

We should slowly pull those constants in and use them when/where we can.

Reply:
👍  Added the constant here.